users/views.py

```diff
@@ -35,14 +32,75 @@ def post(self, request):
 
         user, created = User.objects.get_or_create(
             kakao_id  = kakao_number,
-            defaults  ={
-                'nickname'  : nick_name,
-                'email'     : email,
-                'image_url' : profile_image
+            defaults  = {
+                'nickname' :nick_name,
+                'email'    :email,
+                'image_url':profile_image
             }
         )
 
         kakao_user = User.objects.get(kakao_id=kakao_number)
         token      = jwt.encode({"id": kakao_user.id, 'exp':datetime.utcnow() + timedelta(days=2)}, SECRET_KEY, algorithm=ALGORITHM)
 
-        return JsonResponse({"TOKEN": token}, status=200)
\ No newline at end of file
+        return JsonResponse({"TOKEN": token}, status=200)
+
+@login_decorator
+class BankAccountView(View):
+    def post(self, request, user_id):
+        try:
+            data           = json.loads(request.body)
+            account_holder = data["account_holder"]
+            account_number = data["account_number"]
+
+            if not User.objects.filter(id=user_id).exists(): 
+                return JsonResponse({"ERROR" : "USER_DOES_NOT_EXIST"}, status=404)
+            
+            user         = User.objects.get(id=user_id)
+            user_account = user.bank_account
+
+            for i in account_number:
+                if i == "-":
+                    return JsonResponse({"ERROR":"DO_NOT_ENTER_STRING"}, status=400)
```

Comment:
불필요한 for 문을 없애고 아래와 같이 수정해줄 수 있겠네요.
```suggestion
            if "-" in account_number:
                return JsonResponse({"ERROR":"DO_NOT_ENTER_STRING"}, status=400)
```

Reply:
수정사항 반영하였습니다.